frontend/app/src/components/taxreport/TaxReportOverview.vue

```diff
@@ -15,11 +15,7 @@
               <tr v-for="(item, key) in overview" :key="key">
                 <td>{{ key | splitOnCapital }}</td>
                 <td>
-                  {{
-                    item
-                      | calculatePrice(exchangeRate(currency.ticker_symbol))
-                      | formatPrice(floatingPrecision)
-                  }}
+                  <amount-display :amount="item"></amount-display>
```

Comment:
This will only display the value but it, won't do the price conversion calculation right? it also needs the  `fiat-currency="USD"` so that it converts the value to the selected currency.

Reply:
Give me a moment to clarify that though before doing anything

Reply:
Ignore that comment apparently that was a bug.